engine/class_modules/sc_demon_hunter.cpp

```diff
@@ -5198,6 +5214,13 @@ struct fel_rush_t : public demon_hunter_attack_t
     // Fel Rush and VR shared a 1 second GCD when one or the other is triggered
     if ( p()->cooldown.movement_shared->down() )
       return false;
+   
+    // Not usable while Dash of Chaos buff is present and is locked out for 1 second upon buff expiration
+    if ( p()->talent.havoc.dash_of_chaos && p()->buff.dash_of_chaos && p()->buff.dash_of_chaos->check())
```

Comment:
`p()->talent.havoc.dash_of_chaos` needs to be `p()->talent.havoc.dash_of_chaos->ok()`

Reply:
This has been fixed in most recent commit